.github/workflows/ci-test-tlm.yml

```diff
@@ -42,7 +42,7 @@ jobs:
         run:  cleanlab login --key "${{ secrets.CLEANLAB_STUDIO_CI_API_KEY }}"
       - name: Run tests
         run: |
-          pytest tests/tlm/ --ignore=tests/tlm/test_properties.py --verbose
+          pytest --ignore=tests/tlm/test_properties.py --verbose
```

Comment:
```suggestion
          pytest tests/tlm/ --ignore=tests/tlm/test_properties.py --verbose
```

Reply:
how does this keep getting erased :o wild